.github/workflows/validate.yml

```diff
@@ -0,0 +1,32 @@
+# Reference on this file: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
```

Comment:
This workflow looks quite similar to the "tests.yml" one. Maybe we could merge them together?

Reply:
Done!